src/docs/metrics/sentiment-metrics/weighted-sentiment-metrics/index.md

```diff
@@ -53,6 +53,18 @@ sentiment has a strong positive (negative) polarity but with a low volume, the
 **Sentiment Weighted** metric won't have significant changes and will
 stay around 0.
 
+We have `5m`, `1h`, and `1d` weighted sentiment metrics. The difference between them is in the metrics that we use to calculate them.
+- for sentiment_weighted (5m) we use sentiment_balance (5m) and unique_social_volume (5m)
+- for sentiment_weighted_1h we use sentiment_balance (1h) and unique_social_volume_1h
+- for sentiment_weighted_1d we use sentiment_balance (1d) and unique_social_volume_1d
+
+The difference between base metrics lies in the interval on which we aggregate these metrics.
+In case of `5m` aggregated metrics when we calculate standard devIation and average for a window $d$ we take $d$ 5m's datapoints.
+
+In case of `1h` aggregated metrics -- we take $d$ 1h's datapoints.
+
+In case of `1d` aggregated metrics -- we take $d$ 1d's datapoints.
```

Comment:
I don't fully understand `we take d 1h's datapoints`. What does this mean?

Reply:
I want to say that we aggregate data to `1h` intervals then calculate std and avg on the window with the size of d on this data.

Reply:
I mean that the wording/grammar sounds strange to me.